### PR TITLE
20.03 mutter backports

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
```
jtojnar: should we backport a fix for GNOME Shell freezing after 50 days of idle to 19.09? https://mail.gnome.org/archives/distributor-list/2020-April/msg00001.html
```

This will do the above ^ plus other stuff that's fine for the stable channel.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by worldofpeace** [comment]
@GrahamcOfBorg test gnome3-xorg gnome3

**Comment by worldofpeace** [comment]
@GrahamcOfBorg test pantheon

File: pkgs/desktops/gnome-3/core/mutter/default.nix [hunk 1]
```diff
@@ -111,6 +111,13 @@ stdenv.mkDerivation rec {
       sha256 = "0imy2j8af9477jliwdq4jc40yw1cifsjjf196gnmwxr9rkj0hbrd";
     })
 
+    # gnome-3-34 2020-04-24
+    # also fixes https://mail.gnome.org/archives/distributor-list/2020-April/msg00001.html
```
Comment (jtojnar) [inline pkgs/desktops/gnome-3/core/mutter/default.nix:115]: This has already been fixed in 3.34.2:

https://gitlab.gnome.org/GNOME/mutter/-/commit/f4574ea36f0b2babe21b3ac1ea3c801ca32bc543

Comment (worldofpeace) [reply pkgs/desktops/gnome-3/core/mutter/default.nix]: comment removed

**Review by jonringer — APPROVED** [review]
LGTM

just don't see the `cherry-pick -x` message
```
https://github.com/NixOS/nixpkgs/pull/85966
31 package built:
adapta-gtk-theme chrome-gnome-shell gnome3.gnome-control-center gnome3.gnome-session gnome3.gnome-shell gnome3.gnome-terminal gnome3.gnome-tweak-tool gnome3.gpaste gnome3.mutter gnome3.mutter328 gnome3.pomodoro gnomeExtensions.gsconnect monitor pantheon.elementary-greeter pantheon.elementary-gsettings-schemas pantheon.elementary-session-settings pantheon.gala pantheon.switchboard-plug-pantheon-shell pantheon.switchboard-with-plugs pantheon.wingpanel pantheon.wingpanel-applications-menu pantheon.wingpanel-indicator-bluetooth pantheon.wingpanel-indicator-datetime pantheon.wingpanel-indicator-keyboard pantheon.wingpanel-indicator-network pantheon.wingpanel-indicator-nightlight pantheon.wingpanel-indicator-notifications pantheon.wingpanel-indicator-power pantheon.wingpanel-indicator-session pantheon.wingpanel-indicator-sound pantheon.wingpanel-with-indicators
```

**Comment by worldofpeace** [comment]
> just don't see the cherry-pick -x message

These are divergent from master also.

**Pull request merged**